### PR TITLE
add x-amz-bucket-region header on bucket HEAD response

### PR DESCRIPTION
**Pull request by watsaqat**

### CODE REVIEW

File: lib/api/bucketHead.js [hunk 1]
```diff
@@ -30,7 +30,8 @@ function bucketHead(authInfo, request, log, callback) {
             authInfo,
             bucket: bucketName,
         });
-        return callback(null, corsHeaders);
+        const headers = { 'x-amz-bucket-region': bucket._locationConstraint };
```
Comment (tmacro) [inline lib/api/bucketHead.js]: logic looks good, however the model provides a getter for this.
https://github.com/scality/Arsenal/blob/6347358cc22c4f2703f0ff309b006f1842bd1e40/lib/models/BucketInfo.js#L535

**Review by tmacro — CHANGES_REQUESTED** [review]

File: lib/api/bucketHead.js (hunk 1 above)
Comment (watsaqat) [reply lib/api/bucketHead.js]: thanks, didn't find it at first

**Review by jonathan-gramain — APPROVED** [review]
Whenever you're ready to have the code merged, you can squash the commits into one, add the ticket number in the commit summary, before doing `/approve`

**Review by tmacro — APPROVED** [review]

**Pull request merged**